config/package-bundle/config/values-schema.yml

```diff
@@ -0,0 +1,19 @@
+#@data/values-schema
+---
```

Comment:
Hey @praveenrewar so I had a similar conversation with yash about these values.

It would be ideal if we could align them to what TCE were using:

https://github.com/carvel-dev/kapp-controller/pull/1047/files#r1066411323

Reply:
Sorry, missed that earlier conversation, I will add the other values as well.

Reply:
I have started working on adding the overlays for these additional values, but I feel this should be part of another follow up PR?